assignment.hs

```diff
@@ -23,40 +23,40 @@ import Prelude
 -- returns the reciprocal of that number. This should make (1a) and (1b) 
 -- correct in the main function at the end
 
-reciprocal x = . . .
+reciprocal x = 1 / x
 
 
 -------
 -- 2 --
 -------
 
--- write a function square that squared a number. Uncomment (2) below
+-- write a function square that squares a number. Uncomment (2) below
 -- and check it works.
 
--- your code here
+square x = x*x
 
 -------
 -- 3 --
 -------
 
 -- use the two previous functions to write a new function, `oneOverXSquared`,
 -- that returns the reciprocal of the square of its parameter
--- (that it, 1 / x²). Make sure (3) works.
+-- (that is, 1 / x²). Make sure (3) works.
 
--- your code
```

Comment:
We haven't seen this in class, but you can also write

~~~haskell
oveOverXSquared = reciprocal . square
~~~

That creates a new function that is the composition of the originals.